addons/bus/static/tests/bus_test_helpers.js

```diff
@@ -26,6 +32,87 @@ export function defineBusModels() {
 
 export const busModels = { BusBus, IrWebSocket };
 
+export async function startServer() {
+    const { env } = await makeMockServer();
+    const pyEnv = env;
+    pyEnv["res.users"].write([serverState.userId], {
+        groups_id: pyEnv["res.groups"]
+            .search_read([["id", "=", serverState.groupId]])
+            .map(({ id }) => id),
+    });
+    return env;
+}
+
+const stepState = {
```

Comment:
The whole point is to clean up the bus module. I would rather not add all those test helpers just for tests that will soon be moved back to mail. If multiple modules are using it, it should be in the web module.

Reply:
I'm not sure I understand, if we move this code (which comes originally from `mail`) to `web` than it could be used by both `bus` and `mail` but all the imports in `mail` would need to be changed, wouldn't they?

Reply:
Ideally, yes, but to avoid blocking your progress with a heavy diff, you could import it from the new web location and continue exporting it as before from `mail_test_helpers`.